Makefile

```diff
@@ -48,6 +48,6 @@ repl:
 		swift -F '$(TMPDIR)/SQLite.swift/Build/Products/Debug'
 
 sloc:
-	@zsh -c "grep -vE '^ *//|^$$' SQLite/*.{swift,h,c} | wc -l"
+	@zsh -c "grep -vE '^ *//|^$$' SQLite/*/*.{swift,h,m} | wc -l"
```

Comment:
I tried "make sloc" and only got errors, and this made it work.  (_I almost deleted everything from the Makefile to "start fresh", but didn't want to deal with the potential repercussions as I'd already changed so much!  (cough)bike shedding(/cough)_  )